Base.rte/Devices/Shared/Scripts/RecoilTilt.lua

```diff
@@ -1,16 +1,16 @@
 function Create(self)
 	self.setAngle = 0;
-	self.tilt = 0.2;
+	self.tilt = 0.5 / math.sqrt(self.Radius);
```

Comment:
The idea here is to make bigger actors have less recoil right? I wonder if it should have been Mass instead of Radius? I guess it's fine either way.

Reply:
"Self" is the gun, not an actor :p

It is based on Radius because larger weapons create a higher arc when rotating, which I'm trying to diminish.

Reply:
Ahh gotcha, that makes sense. Yeah, I just skimmed it so I didn't realize self was the gun not the actor, derp.